apiserver/objects_test.go

```diff
@@ -219,11 +326,276 @@ func (s *putCharmObjectSuite) TestUploadBumpsRevision(c *gc.C) {
 	c.Assert(sch.BundleSha256(), gc.Not(gc.Equals), "")
 }
 
+func (s *putCharmObjectSuite) TestUploadVersion(c *gc.C) {
+	expectedVersion := "dummy-146-g725cfd3-dirty"
```

Comment:
There is no need to keep using the term "dummy" in this context. "testcharm" or the like is equally descriptive and preferable. Especially in places which don't require large scale changes like here. 

I understand that fixing `testcharms.Repo.CharmArchivePath(c.MkDir(), "dummy")` in the tests is a bigger undertaking.

Reply:
I have changed over every occurrence that wouldn't require a larger scale change. it's only really calls to `testcharms.Repo.CharmArchive` which remain using `dummy`